syft/frameworks/torch/hook.py

```diff
@@ -5,7 +5,7 @@
 import types
 import syft
 from syft import workers
```

Comment:
Do these on separate lines to avoid merge conflicts

Reply:
I don't really understand this :D I only see separated lines.

Reply:
Yeah I'm not sure either

Reply:
Sorry about this, meant the line after. Doing these comments on mobile is difficult